### DISCUSSION TITLE
窗口是乱码的

### DISCUSSION THREAD

**Discussion by mxyhhhhh111** [hjenryin/BCJH-Metropolis#202] [Q&A]
我的跑完为啥全部都是乱码
![image](https://github.com/hjenryin/BCJH-Metropolis/assets/115398538/39dd5091-8dbc-4fa1-b661-b7218045d577)

**Comment by hjenryin** [comment]
不是很清楚……我试着改一下吧，不过不确定可不可以解决

**Reply by mxyhhhhh111** [reply]
![image](https://github.com/hjenryin/BCJH-Metropolis/assets/115398538/c9dda225-e512-4228-aa6c-8dbf3350ce02)
只有生成的部分是乱码，win11，哭死

**Reply by hjenryin** [reply]
诶 怎么和之前不一样了🤔

**Reply by hjenryin** [reply]
我大概有个想法，回头传一个版本你可以再试一下

**Reply by hjenryin** [reply]
失败了，我也不知道咋办😭

**Reply by hjenryin** [reply]
我现在能知道这个问题是我的程序是gbk编码但输出是utf-8，但奇怪的是我在程序里面已经设置了输出gbk，不知为何没有生效，我也无法复现你的问题🥺

**Reply by mxyhhhhh111** [reply]
之前也是这样的，刚开始没截下面的部分

**Reply by mxyhhhhh111** [reply]
好滴好滴，感谢大佬了！
